packages/rum-angular/src/index.ts

```diff
@@ -0,0 +1,83 @@
+/**
+ * MIT License
+ *
+ * Copyright (c) 2017-present, Elasticsearch BV
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in
+ * all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+ * THE SOFTWARE.
+ *
+ */
+
+import {
+  Router,
+  Event,
+  NavigationStart,
+  NavigationEnd,
+  NavigationError
+} from '@angular/router'
+import { Injectable } from '@angular/core'
+import { Promise } from 'es6-promise'
+import { init as apmInit, apm } from '@elastic/apm-rum'
+
+@Injectable({
+  providedIn: 'root'
+})
+export class ApmService {
+  constructor(public router: Router) {}
+
+  init(config) {
+    apmInit(config)
+    /**
+     * Start listening to route change once we
+     * intiailize to set the correct transaction names
+     */
```

Comment:
TODO: We need the observer to be disabled when the agent is inactive. Will be added by https://github.com/elastic/apm-agent-rum-js/issues/386

Reply:
Made changes to respect the active flag and added tests.